docs/hdl.rst

```diff
@@ -2,10 +2,43 @@ Hardware Description Languages
 ------------------------------
 
 The two most common **H**\ ardware **D**\ escription **L**\ anguages are
-Verilog and VHDL (the toolchain we are using only supports Verilog).
+Verilog and VHDL.
+
+.. NOTE:: The pre-built toolchain we are releasing supports Verilog only.
+  However, `GHDL <https://github.com/ghdl>`_ might be used as a VHDL frontend
+  for Yosys. See `ghdl-yosys-plugin <https://github.com/ghdl/ghdl-yosys-plugin>`_.
+
+When writing HDL, a tool called ``yosys`` is used to convert the
```

Comment:
I think a lot of this information probably belongs in a separate section?

Reply:
Agree. However, note that all I did was move it from `verilog.rst` to `hdl.rst`. Currently, that's shown in https://workshop.fomu.im/en/latest/verilog.html

Should it be moved to https://workshop.fomu.im/en/latest/background.html#turning-code-into-gates instead?